src/frontend/src/icp/components/transactions/IcTransaction.svelte

```diff
@@ -19,19 +19,13 @@
 
 	export let transaction: IcTransactionUi;
 
-	let transactionType: IcTransactionType;
+	let type: IcTransactionType;
```

Comment:
isn't `type` some sort of reserved TypeScript keyword? I would bet that's why `transactionType` was used.

Reply:
i thought so, but we use type in ETH and BTC too

```typescript
let type: TransactionType;

let type: EthTransactionType;
```

so, maybe I shall rename those and leave this one as it is?

Reply:
Sounds good then!